lib/webdrivers/iedriver.rb

```diff
@@ -35,7 +41,7 @@ def downloads
           items = doc.css("Key").collect(&:text)
           items.select! { |item| item.include?('IEDriverServer_Win32') }
           ds = items.each_with_object({}) do |item, hash|
-            key = normalize item[/^[^\/]+/]
+            key = normalize item[/\d+\.\d+\.\d+(?=\.zip$)/]
```

Comment:
I'm thinking we want to avoid the digit formatting and grab everything before the `_` like: `item[/([^_]+)\.zip/, 1]`